tests/unit/output/test_local.py

```diff
@@ -20,6 +21,29 @@ def test_save_missing(self):
             with self.assertRaises(o.DoesNotExistError):
                 o.save()
 
+    def test_str_if_inside_repo(self):
+        o = self._get_output()
+        self.assertEqual("path", str(o))
+
+    def test_str_if_workdir_is_inside_repo(self):
+        stage = Stage(self.dvc)
+        work_dir = os.path.join(stage.repo.root_dir, "models")
+        os.makedirs(work_dir, exist_ok=False)
```

Comment:
`exist_ok` does not exist in Python2.7

Reply:
@skshetry there is `compat` library in `dvc.utils`

Reply:
Found on https://github.com/iterative/dvc/blob/6147b93d24f2f508a8ec3db40f10a77bf85b6a71/dvc/utils/compat.py#L74

Reply:
you can use `utils.makedirs` :)